test/NinjaTools.FluentMockServer.API.Exclude.Tests/Extensions/ProxyExtensionTests.cs

```diff
@@ -32,20 +31,16 @@ public void CanHaveContentBasedOnMethod_Should_Return_Value_Based_On_Http_Specif
             // Arrange
             var sut = new DefaultHttpContext().Request;
             sut.Method = method.ToString();
-            
+
             // Act && Assert
             sut.CouldHaveContentBasedOnMethod().Should().Be(expected);
         }
-        
-        
-        
     }
 
     [SuppressMessage("ReSharper", "InvokeAsExtensionMethod")]
     public class HttpRequestMessageFactoryTests
     {
-
-        private DownstreamContext CreateContext([NotNull] HttpContext httpContext)
+        private DownstreamContext CreateContext([JetBrains.Annotations.NotNull] HttpContext httpContext)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'CreateContext'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)